content/blog/create-a-cold-tiered-volume-and-use-hpe-ezmeral-data-fabric-object-store-as-remote-target.md

```diff
@@ -0,0 +1,656 @@
+    "Statement":
+    [
+        {
+            "Effect": "Allow",
+            "Action":
+            [
+                "s3:ListAllMyBuckets",
+                "s3:CreateBucket",
+                "s3:GetBucketLocation"
+            ],
+            "Resource":
+            [
+                "arn:aws:s3:::*"
+            ]
+        },
+        {
+            "Effect": "Allow",
+            "Action":
+            [
+                "s3:ListBucket",
+                "s3:GetBucketLocation"
+            ],
+            "Resource": "arn:aws:s3:::s3-test-iam-user-bucket/*"
+        },
+        {
+            "Effect": "Allow",
+            "Action":
+            [
+                "s3:PutObject",
+                "s3:PutObjectAcl",
+                "s3:GetObject",
+                "s3:GetObjectAcl",
+                "s3:DeleteObject"
+            ],
+            "Resource": "arn:aws:s3:::s3-test-iam-user-bucket/*"
+        }
+    ]
+}
+```
+
+To associate an IAM Policy to an IAM User:
+
+```shell
+sudo -u mapr /opt/mapr/bin/mc admin policy set s3-admin-alias PolicyPublicRead users='s3-test-iam_user' account='s3test' domain='primary'
+sudo -u mapr /opt/mapr/bin/mc admin policy set s3-admin-alias GrantBucketOperations users='s3-test-iam_user' account='s3test' domain='primary'
+```
+
+#### Create a Bucket for the IAM User
+
+First, you need to generate the access key and secret key for the IAM User - "s3-test-iam_user".
+
+```shell
+sudo -u mapr maprcli s3keys generate -domainname primary \
+  -accountname s3test \
+  -username 's3-test-iam_user'
+```
+
+☝Then you would get the access key and secret key for the IAM User - "s3-test-iam_user".
+
+```shell
+sudo -u mapr mc alias set s3-test-iam_user-alias https://`hostname -f`:9000 \
+{ACCESS_KEY} \
+{SECRET_KEY} \
+--api "s3v4" --path "off" --json
+
+sudo -u mapr mc mb --account s3test --ignore-existing --disable-versioning --json s3-test-iam_user-alias/s3-test-iam-user-bucket
+```
+
+☝Now you have created a Bucket named "s3-test-iam-user-bucket" using the IAM User - "s3-test-iam_user".
+Because "s3-test-iam_user" is inside account - "s3test", the Bucket will be also placed under account - "s3test".
+
+To list Buckets using the `mc` command:
+
+```shell
+/opt/mapr/bin/mc ls --account s3test --versions --recursive --summarize --json s3-test-iam_user-alias
+```
+
+Sample output:
+
+```json
+{
+ "status": "success",
+ "type": "folder",
+ "lastModified": "2023-02-16T15:22:16+08:00",
+ "size": 23897893980,
+ "key": "s3-test-iam-user-bucket/",
+ "etag": "",
+ "url": "https://m2-maprts-vm197-172.mip.storage.hpecorp.net:9000/",
+ "versionOrdinal": 1
+}
+{
+"totalObjects": 1,
+"totalSize": 23897893980
+}
+```
+
+#### Install the AWS CLI and put an file into the Bucket
+
+To install the AWS CLI, refer to this Amazon AWS document 👉 [Installing or updating the latest version of the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
+
+Then you create a profile for the IAM User:
```

Comment:
Take out the word "you" so it just reads "Then, create a profile for the IAM User:"